<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CannyEdge</class>
 <widget class="QWidget" name="CannyEdge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="window">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>314</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>window</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>109</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Image Type</string>
   </property>
   <widget class="QRadioButton" name="image_color">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="image_grey">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>25</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grey</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>21</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>Value</string>
   </property>
   <widget class="QSlider" name="valueSild">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>valueSild</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
